RELEASENOTES.md

```diff
@@ -7,6 +7,141 @@ This document contains the release notes for `rippled`, the reference server imp
  
 Have new ideas? Need help with setting up your node? [Please open an issue here](https://github.com/xrplf/rippled/issues/new/choose).
 
+# Introducing XRP Ledger version 1.12.0
+
+Version 1.12.0 of `rippled`, the reference server implementation of the XRP Ledger protocol, is now available. This release adds new features and bug fixes, and introduces these amendments:
+
+- `featureAMM`
```

Comment:
I'm not sure if we should include the `feature` prefix. For [feature](https://xrpl.org/feature.html), when specifying the name in the `feature` field, should the `feature` prefix be included?

Reply:
The `feature` prefix is _not_ part of the amendment name.

```suggestion
- `AMM`
```